### DISCUSSION TITLE
Has there been a change in Mergify that causes to drop approvals after rebase?

### DISCUSSION THREAD

**Discussion by nixpanic** [Mergifyio/mergify#2939] [Q&A]
Hi!

Over at Ceph-CSI we are big fans of Mergify, thanks :+1:

Unfortunately, it seems that there are occasional approvals dropped after a `@mergify rebase` or automatic rebase before automatic merging. This is very sad, as reviewing changes is one of the main time-consuming tasks of the maintainers.

We have an example at https://github.com/ceph/ceph-csi/pull/2307#issuecomment-883949567

This most likely started to happen somewhere last week. If there has been an intentional change, we are happy to adjust the configuration. But, of course we also understand that some bugs sneak through, and if that is the case, we can open a GitHub issue with the details shared in this discussion (and more if you tell us what you need).

Thanks!
Niels

**Comment by jd** [comment]
Hi @nixpanic 

This should not happen; we think it's a bug and we'll track it here https://github.com/Mergifyio/mergify-engine/issues/2940

Thanks for reporting!

**Reply by nixpanic** [reply]
Thanks for the reply!

It seems you guys already fixed the issue too, great job :cake:
